<?xml version="1.0"?>
<launch>
  <node pkg="brushee_mppi_path_tracker" type="brushee_mppi_path_tracker_node" name="brsuhee_mppi_path_tracker" output="screen">
    <param name="vx_min" value="-1.0"/>
    <param name="horizon" value="20"/>
    <param name="num_samples" value="500"/>
    <param name="vel_weight" value="1.0"/>
    <param name="path_weight" value="2.0"/>
    <param name="angle_weight" value="1.0"/>
  </node> 
</launch>